lib/features/create-append-anything/CreateAppendEditorActions.js

```diff
@@ -25,16 +25,25 @@ CreateAppendEditorActions.prototype.registerActions = function() {
   var selection = this._injector.get('selection', false);
   var contextPad = this._injector.get('contextPad', false);
   var palette = this._injector.get('palette', false);
+  var popupMenu = this._injector.get('popupMenu', false);
 
   const actions = {};
 
   // append
   if (selection && contextPad) {
     assign(actions, {
       'appendElement': function(event) {
-        contextPad.triggerEntry('append', 'click', event);
-      } }
-    );
+        const selected = selection && selection.get();
+
+        if (selected.length > 1) {
```

Comment:
If we generalize it, let's generalize it the full way:

This part should not return, but trigger create.